int64 x
int64 y
int64 w
int64 h
float64 angle
int64 label
float64 score